components/inputs/input-time.js

```diff
@@ -144,47 +144,47 @@ class InputTime extends LitElement {
 		const input = html`
 			<label>
 				<span class="${this.label && !this.labelHidden ? 'd2l-input-label' : 'd2l-offscreen'}" id="${this._dropdownId}-label">${this.label}</span>
```

Comment:
Having a `<label>` that doesn't use `for` and doesn't wrap its input isn't useful, so I think the `<span>` can just become the main node here.

Reply:
Whoops - absolutely, thanks :)